app/views/accounts/_emails.html.erb

```diff
@@ -1,17 +1,8 @@
 <div class="clearfix border-bottom border-primary-light">
   <div class="col col-12 margin-bottom-1 margin-top-0">
-    <h3 class="h2 col col-6 margin-0">
+    <h3 class="h2 col col-12 margin-0">
       <%= t('account.index.email_addresses') %>
     </h3>
-    <div class="right-align col col-6">
-      <% if EmailPolicy.new(current_user).can_add_email? %>
```

Comment:
Should this stay? This affects adding emails instead of phone numbers.

Reply:
From our discussion with UI/UX team they wanted those to be removed. They are not 508 compliant and we have a link in the leftnav bar that allows you to add emails.

Reply:
In that case, would we want to remove them from all account pages? I think I'd also expect the `prefix_with_plus` helper to be removed, since I believe it's only used with these MFA add buttons.

Reply:
After team discussion its decided to address all of them in LG-5591. So we will just remove from this page for now.